docs/csharp/quick-starts/index.md

```diff
@@ -44,3 +44,10 @@ different actions.
 
 This beginner lesson assumes that you have finished the [Hello World](hello-world.yml) and
```

Comment:
This is a nit, but at least to me, "beginning" sounds much better than "beginner"